test/test_kms_with_cli.py

```diff
@@ -132,9 +132,24 @@ def no_bool_convert(pairs):
 
 def test_GenerateQuote_and_VerifyQuote(base_url, headers):
     print('====================test_GenerateQuote_and_VerifyQuote start===========================')
+    mrenclave = '870c42c59bc74c7ad22869411709e4f78ac3c76add6693bb43296b03362e5038';
+    mrsigner = 'c30446b4be9baf0f69728423ea613ef81a63e72acf7439fa0549001fd5482835';   
```

Comment:
need a note for original data here as well.

Reply:
will be add